### PR TITLE
Adding latest catalog from Tessa

### PR DESCRIPTION
**Pull request by GypsyBojangles**
Integrating Tessa's changes for DEAfrica map catalog structure and product names.

### CODE REVIEW

File: terria/africa.json [hunk 1]
```diff
@@ -235,86 +234,97 @@
...
+               "linkedWcsCoverage": "ls_usgs_fc_scene",
+               "ignoreUnknownTileErrors": true,
+               "opacity": 1,
+               "featureTimesProperty": "data_available_for_dates"
             }
          ]
       },
       {
-         "name": "Ancillary Data",
+         "name": "Elevation",
```
Comment (alexgleith) [inline terria/africa.json:315]: The reason for calling this ancillary is that we may include other ancillary data here.

For example, the external surface water dataset may go here?

File: terria/africa.json [hunk 2]
```diff
@@ -5,97 +5,122 @@
...
-                        "template": "<table>  <tr><td><b>Time</b></td><td>{{time}}</td></tr>  <tr><td><b>Blue - 490</b></td><td>{{bands.blue}}</td></tr>  <tr><td><b>Green - 560</b></td><td>{{bands.green}}</td></tr>  <tr><td><b>Red - 660</b></td><td>{{bands.red}}</td></tr>  <tr><td><b>Near Infrared (NIR) - 840</b></td><td>{{bands.nir}}</td></tr>  <tr><td><b>Shortwave Infrared (SWIR) - 1650</b></td><td>{{bands.swir1}}</td></tr>  <tr><td><b>Shortwave Infrared (SWIR) - 2220</b></td><td>{{bands.swir2}}</td></tr>  <tr><td><b>NDVI - Red, NIR</b></td><td>{{band_derived.ndvi}}</td></tr>  <tr><td><b>NDWI - Green, SWIR</b></td><td>{{band_derived.ndwi}}</td></tr></table><p><chart id='{{lat}}{{lon}}{{time}}' title='NBART at {{lat}},{{lon}}' column-units='Wavelength (nm), Reflectance x 10k' preview-x-label='Surface Reflectance'>nm,Surface Reflectance\n490,{{bands.blue}}\n560,{{bands.green}}\n660,{{bands.red}}\n840,{{bands.nir}}\n1650,{{bands.swir1}}\n2220,{{bands.swir2}}</chart></p><p><b>Imagery available for dates:</b>{{#data_available_for_dates}}<br/>{{.}}{{/data_available_for_dates}}</p>"
-                     }
-                 }
+                     "name": "Annual median",
```
Comment (alexgleith) [inline terria/africa.json:98]: Not sure I agree with changing it to just 'median'.

I think a cloud-free mosaic is more widely understood, but failing that, the "Annual geomedian" is correct. Or "Annual geometric median".

Comment (GypsyBojangles) [reply terria/africa.json:98]: From Slack:"
Thanks Alex appreciate the input, these changes were done with input from Fang, Negin and Chad, and Adam reviewed. We've also done work on rewriting the meta data for each, so for where we've simplified naming we have explained these in more detail within the data descriptions"

File: terria/africa.json (hunk 1 above)
Comment (GypsyBojangles) [reply terria/africa.json:315]: From Slack:"
Thanks Alex appreciate the input, these changes were done with input from Fang, Negin and Chad, and Adam reviewed. We've also done work on rewriting the meta data for each, so for where we've simplified naming we have explained these in more detail within the data descriptions"

**Comment by GypsyBojangles** [comment]
You can validate this file by:
- veriying its JSON syntax in an editor
- clicking and draggin the file onto: https://maps.digitalearth.africa/#clean

File: terria/africa.json (hunk 1 above)
Comment (cbur24) [reply terria/africa.json:315]: The general idea with the naming of these top-level headings is to group datasets by themes. So you'd have a `Water` heading where anything relating to water would go, a `Satellite Images` section where all surface reflectance goes, etc.  Using this convention, it makes sense to have an `Elevation` heading where all datasets with height data can go.

**Pull request merged**